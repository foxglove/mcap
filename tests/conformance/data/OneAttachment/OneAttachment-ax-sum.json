{
  "records": [
    {"type": "Header", "fields": [["library", ""], ["profile", ""]]},
    {
      "type": "Attachment",
      "fields": [
        ["create_time", "1"],
        ["data", ["1", "2", "3"]],
        ["log_time", "2"],
        ["media_type", "application/octet-stream"],
        ["name", "myFile"]
      ]
    },
    {"type": "DataEnd", "fields": [["data_section_crc", "3051995293"]]},
    {
      "type": "AttachmentIndex",
      "fields": [
        ["create_time", "1"],
        ["data_size", "3"],
        ["length", "78"],
        ["log_time", "2"],
        ["media_type", "application/octet-stream"],
        ["name", "myFile"],
        ["offset", "25"]
      ]
    },
    {
      "type": "SummaryOffset",
      "fields": [
        ["group_length", "87"],
        ["group_opcode", "10"],
        ["group_start", "116"]
      ]
    },
    {
      "type": "Footer",
      "fields": [
        ["summary_crc", "4117354065"],
        ["summary_offset_start", "203"],
        ["summary_start", "116"]
      ]
    }
  ],
  "meta": {"variant": {"features": ["ax", "sum"]}}
}
