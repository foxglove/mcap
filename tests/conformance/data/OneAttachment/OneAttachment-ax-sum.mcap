version https://git-lfs.github.com/spec/v1
oid sha256:48af102ee5043f74dd61ffe85cc0ca4aef4e20a3a476b1a455db9037bbad1df5
size 310
