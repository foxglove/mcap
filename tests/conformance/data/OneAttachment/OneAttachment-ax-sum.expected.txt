Header library="" profile=""
Attachment content_type=application/octet-stream created_at=1 data=<010203> log_time=2 name=myFile
AttachmentIndex content_type=application/octet-stream data_size=3 length=78 log_time=2 name=myFile offset=25
SummaryOffset group_length=0 group_opcode=12 group_start=116
SummaryOffset group_length=79 group_opcode=9 group_start=116
SummaryOffset group_length=0 group_opcode=7 group_start=195
Footer summary_offset_start=195 summary_start=116
